### ISSUE TITLE
Please update docs to note that RancherOS does not support script user-data.

### ISSUE THREAD

**Issue by allingeek** [rancher/os#2229]

**Comment by niusmallnan** [comment]
Not sure which datasource you mean, we have this description:
```
When you create a RancherOS instance on AWS, for example, you can optionally provide cloud-config passed in the user-data field. Inside the RancherOS instance, cloud-init process will retrieve the cloud-config content through its AWS cloud-config data source, which simply extracts the content of user-data received by the VM instance. If the file starts with “#cloud-config”, cloud-init will interpret that file as a cloud-config file. If the file starts with #!<interpreter> (e.g., #!/bin/sh), cloud-init will simply execute that file. You can place any configuration commands in the file as scripts.
```
Refer to http://rancher.com/docs/os/v1.1/en/configuration/

**Comment by SvenDowideit** [comment]
@allingeek I think you need to be alot more specific. I was developing and fixing issues for cloud-init scripts like https://github.com/SvenDowideit/bin-dir/blob/master/cloud-configs/airgap/install-node.sh on RancherOS versions from before the LTS releases. There are fixes in 1.1 to deal with issues with proxying, but it has supported scripted user-data for a very long time.

**Comment by janeczku** [comment]
@allingeek Could you please provide:
- RancherOS versions
- cloud-init datasource (e.g. ec2)
- content of userdata that you used
